plugins/plugin-debugger/che-plugin-debugger-ide/src/main/java/org/eclipse/che/plugin/debugger/ide/debug/DebuggerPresenter.java

```diff
@@ -537,4 +545,9 @@ public void onFailure(Throwable caught) {}
               public void onSuccess(VirtualFile result) {}
             });
   }
+
+  @Override
+  public void onWorkspaceStopped(WorkspaceStoppedEvent event) {
+    resetView();
```

Comment:
I realized we should also clear the breakpoint list.
`view.setBreakpoints(emptyList())`